src/map/pc.c

```diff
@@ -534,6 +535,17 @@ void pc_rental_expire(struct map_session_data *sd, int i) {
 			break;
 	}
 
+	pc->setreg(sd, script->add_str("@expire_itemid"), nameid);
+	pc->setreg(sd, script->add_str("@expire_refine"), sd->status.inventory[i].refine);
+	pc->setreg(sd, script->add_str("@expire_identify"), sd->status.inventory[i].identify);
+	pc->setreg(sd, script->add_str("@expire_attribute"), sd->status.inventory[i].attribute);
+	pc->setreg(sd, script->add_str("@expire_bound"), sd->status.inventory[i].bound);
+	// pc->setreg(sd, script->add_str("@expire_uniqueid"), sd->status.inventory[i].unique_id);
```

Comment:
I always feel its weird
why it want to use unique_id something like `644253684334719` instead of start from `1` ?
that number is found from my `inventory` table
and the number progress linearly , 1 by 1 with each equipment created ...